### ISSUE TITLE
loading bundles

### ISSUE THREAD

**Issue by artknight** [danilo-valente/jsRequest#2]
it would be great to be able to load bundles

"bundle-name": [
   "script1",  "script2",  "script3",  "script4",  "script5"
]

**Comment by ThePanuto** [comment]
The cool feature of jsRequest is being able to explicitly define which scripts load before others. When loading in a bundle, would the load order be defined by element position in the array?

**Comment by artknight** [comment]
yes, exactly...order the array is preserved when loading. It should also be able to load bundle of bundles. So, for instance

bundle1: [ "script1", "script2", "script3", "script4", "script5" ],
bundle2: [ "script1", "script2", "script3", "script4", "script5" ],
bundle3: [ "bundle1","script1", "script2" ]

This just means that when loading bundle3 I would need all scripts from bundle1 and some additional ones

FYI, up until now I have been using LAB.js, which has all the features but the project is no longer maintained.... and I would rather switch to this one as long as it can offer me the same ( or more ) cepabilities.

**Comment by ThePanuto** [comment]
Today, you can't import array bundled scripts as you specified, but you can import multiple scripts with  `load`. For instance:

    jsRequest
      .load('script1.js', 'script2.js')
      .wait('script3.js', 'script4.js', function() {
          alert('all loaded');
       });

But you idea makes sense. Until we implement it, you could use the above strategy to define load order.

**Comment by danilo-valente** [comment]
@artknight I agree, this is something we should implement. I'm gonna discuss with @ThePanuto the best way to do that.

**Comment by artknight** [comment]
Awesome!
